test/python/test_pack.py

```diff
@@ -224,6 +246,17 @@ def test_deb_pack(deb_archive, tmpdir):
             assert 'chown root:root /usr/lib/tmpfiles.d/{}.conf'.format(project.name) in postinst_script
 
 
+def test_rock_pack(rock_archive):
+    assert zipfile.is_zipfile(rock_archive.filepath), 'rock should be a valid zip-archive'
+    with zipfile.ZipFile(rock_archive.filepath, 'r') as zip_archive:
+        files = zip_archive.namelist()
+        # Tarantool 1.10 and Tarantool 2.2 use Luarocks 2 and Luarocks 3 respectively
```

Comment:
This is very sad, I sure that it would be great to check that archive contains correct name, version and other base information.

Reply:
The package process is different for different luarocks versions and different rockspec content. I check that `cartridge-cli` doesn't fail on `pack rock` and I don't test `luarock pack`

Reply:
Yes, but `cartridge-cli` is responsible for passing correct information to `luarocks`. We sure (hope) that `luarocks` works correctly, so this check means inly checking name and version that `cartridge-cli` passes to it.

Reply:
All our possible actions in case of `rock pack` is to pass correct rockspec to `luarock pack` and validate rockspec/files content before run `luarocks pack` - to show more informative error... But it's an overkill